test/features/caseView.feature

```diff
@@ -27,5 +25,14 @@ Feature: Set of scenarios testing case view functionality
       | Text Field 3        |
       | Text Field 4        |
 
+  @functional
   Scenario: print button visible on case view
-   Then the print button will be visible
+    When a case type containing every field type exists
+    And I create the case
+    Then the print button will be visible
+
+  @functional
+  Scenario: print button not visible on case view
+    When a case with Case Progression functionality exists
```

Comment:
Similar as abive comment `Given a case with the print button not configured exists` would be better here

Reply:
Done.